components/record/MediaActionBar.vue

```diff
@@ -1,35 +1,55 @@
 <template>
-  <section
-    data-qa="action bar"
-    class="d-flex align-items-center"
-  >
-    <b-button
-      v-if="url"
-      :href="url | proxyMedia(europeanaIdentifier)"
-      variant="outline-primary text-decoration-none"
-      data-qa="download button"
-      size="lg"
-      class="mr-3"
-    >
-      {{ $t('actions.download') }}
-    </b-button>
+  <section data-qa="action bar">
+    <b-row>
+      <b-col
+        cols="6"
+        class="d-flex align-items-center"
+      >
+        <b-button
+          v-if="url"
+          :href="url | proxyMedia(europeanaIdentifier)"
+          variant="outline-primary text-decoration-none"
+          data-qa="download button"
+          size="lg"
+          class="mr-3"
+        >
+          {{ $t('actions.download') }}
+        </b-button>
 
-    <SmartLink
-      v-if="rightsStatementIsUrl"
-      :destination="rightsStatement"
-      class="attribution"
-      data-qa="rights statement"
-    >
-      <RightsStatement
-        :rights-statement-url="rightsStatement"
-      />
-    </SmartLink>
-    <span
-      v-else
-      data-qa="rights statement"
-    >
-      {{ rightsStatement }}
-    </span>
+        <SmartLink
+          v-if="rightsStatementIsUrl"
+          :destination="rightsStatement"
+          class="attribution"
+          data-qa="rights statement"
+        >
+          <RightsStatement
+            :rights-statement-url="rightsStatement"
+          />
+        </SmartLink>
+        <span
+          v-else
+          data-qa="rights statement"
+        >
+          {{ rightsStatement }}
+        </span>
+      </b-col>
+      <b-col
+        cols="6"
+        class="d-flex align-items-center justify-content-end"
+      >
+        <span
+          data-qa="provider name"
+          class="is-size-4 view-at"
+        >
+          {{ isShownAt ? $t('actions.viewAt') : $t('actions.providedBy', { provider: providerName }) }}
```

Comment:
When there is no link, the provider name still needs to be marked up with the language code in the lang attribute, on a span element.